src/viewport-observer.js

```diff
@@ -19,11 +19,6 @@ import {toWin} from './types';
 
 /**
  * Returns an IntersectionObserver tracking the Viewport.
- * Only use this if x-origin iframe rootMargin support is considered nice-to-have,
```

Comment:
Note: this comment has been outdated since we removed the extra 25% to viewport checks.